clang/lib/Sema/SemaChecking.cpp

```diff
@@ -14043,28 +14039,63 @@ static bool CheckTautologicalComparison(Sema &S, BinaryOperator *E,
   return true;
 }
 
+namespace {
+struct AnalyzeImplicitConversionsWorkItem {
+  Expr *E;
```

Comment:
Because this is a new interface, can we make it more `const`-correct? Also applies to `ImplicitConversionChecker`. (I don't insist if it turns out to require viral changes.)